docs/backends.md

```diff
@@ -229,4 +228,4 @@ LRS parameters required to connect are:
 
 Optional parameters can be configured if necessary: 
 
-- `headers`: a comma-separated key=value list of LRS server headers
\ No newline at end of file
+- `headers`: a comma-separated key=value list of LRS server headers
```

Comment:
Not sure what the change is here :grin:

Reply:
It's just a newline at the end that my editor puts in automatically since it's the last line in the file and didn't have one.